server/src/main/java/org/apache/druid/segment/loading/RandomStorageLocationSelectorStrategy.java

```diff
@@ -31,9 +34,15 @@
 public class RandomStorageLocationSelectorStrategy implements StorageLocationSelectorStrategy
 {
 
-  private final List<StorageLocation> storageLocations;
+  private List<StorageLocation> storageLocations = null;
```

Comment:
The default values ​​of reference types in Java are all null, so assigning null here should be meaningless.

Reply:
IDEA reports an assignment of null is required due to some customized check rules. Been fixed.